components/layout/Header.module.css

```
@@ -1,22 +1,84 @@
+
+.languageSwitcher {
   display: flex;
+  align-items: center;
+}
+
+.languageLabel {
+  display: flex;
+  padding: 8px 15px;
+  align-items: center;
+  gap: 4.8px;
+  align-items: center;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

align-items is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)